pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -269,8 +291,9 @@ func TestUpdatePod(t *testing.T) {
 	}
 
 	for _, item := range testCases {
+		defer featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, features.PodDisruptionConditions, item.enablePodDisruptionConditions)()
```

Comment:
same here https://github.com/kubernetes/kubernetes/pull/110959/files#r934826728, but seems these tests were already creating multiple controllers in parallel, unless I'm getting it wrong

Reply:
Adjusted